MitigationCopier.py

```diff
@@ -5,11 +5,14 @@
 import datetime
 
 import anticrlf
-from veracode_api_py.api import VeracodeAPI as vapi, Applications, Findings
+from veracode_api_py.api import VeracodeAPI as vapi, Applications, Findings, SCAApplications, Sandboxes
 from veracode_api_py.constants import Constants
 
 log = logging.getLogger(__name__)
 
+ALLOWED_ACTIONS = ['COMMENT', 'FP', 'APPDESIGN', 'OSENV', 'NETENV', 'REJECTED', 'ACCEPTED', 'LIBRARY', 'ACCEPTRISK', 
```

Comment:
this is essentially a constant - doing this as a global variable is OK but constants are better see https://stackoverflow.com/questions/6345840/whats-the-best-way-to-initialise-and-use-constants-across-python-classes